P5/Source/Guidelines/en/ND-NamesDates.xml

```diff
@@ -2269,6 +2085,265 @@ occupation and residence.</p>
+            </person>
+            <person xml:id="p-AM">
+              <persName>Alan Eugene Magee</persName>
+              <birth when="1919-01-13"/>
+              <death when="2003-12-20"/>
+              <ptr type="furtherInfo" target="https://en.wikipedia.org/wiki/Alan_Magee"/>
+            </person>
+            <person xml:id="p-NA">
+              <persName>Nicholas Stephen Alkemade</persName>
+              <birth when="1922-12-10"/>
+              <death when="1987-06-22"/>
+              <ptr type="furtherInfo" target="https://en.wikipedia.org/wiki/Nicholas_Alkemade"/>
+            </person>
+            <person xml:id="p-BG">
+              <persName>Edward Michael Grylls</persName>
+              <birth when="1974-06-07"/>
+              <ptr type="furtherInfo" target="https://www.beargrylls.com/pages/about-bear-grylls"/>
+            </person>
+            <person xml:id="p-JK">
+              <persName>Juliane Koepcke</persName>
+              <birth when="1954-10-10"/>
+              <ptr type="furtherInfo" target="https://en.wikipedia.org/wiki/Juliane_Koepcke"/>
+            </person>
+            <person xml:id="p-JB">
+              <persName>James Boole</persName>
+              <birth>1978, give or take a year</birth>
+              <ptr type="furtherInfo" target="https://www.facebook.com/james.boole/?ref=page_internal"/>
+            </person>
+          </listPerson>
+          <!-- ========= conjunctions and disjunctions ========= -->
+          <altGrp type="location">
+            <alt xml:id="l-KorZ" target="#l-Ken #l-Zam"/>
+          </altGrp>
+        </standOff>
+      </egXML>
+      <p>The <att>where</att> attribute on <gi>event</gi> can point to multiple 
+        canonical place descriptions, which should indicate 
+        that the event took place in multiple locations. For example, the November 2015
+        Paris Attacks took place at six different locations in Paris; this
+        might be encoded as an <gi>event</gi> with a <att>where</att>
+        attribute with six space-separated values, or using a <gi>listPlace</gi> (nested into the <gi>event</gi> or standalone).</p>
+      
+      <p>Events may also be modeled as hierarchies of nested <gi>event</gi>s. For example, a two part meeting might be represented as follows:
+        <egXML xmlns="http://www.tei-c.org/ns/Examples">
+          <event type="meeting" when="2007-05-29">
+            <desc>All day meeting to resolve content models</desc>
+            <event type="preamble" notAfter="13:00:00">
+              <desc>first part</desc>
+            </event>
+            <event type="conclusions" notBefore="13:00:00">
+              <desc>second part</desc>
+            </event>
+          </event>
+        </egXML>
+      </p>
+      
+      <p>For <gi>event</gi> as a child element of <gi>person</gi>, note the Section <ptr target="#NDPERSEpe"/>.</p>
+      <p>
```

Comment:
Maybe I am missing something but I don't understand why `history` and `additional` are mentioned here (and without any explanation/context etc.).

Reply:
Yes, that seems strange to me, too: maybe just a copy-paste error, since it's defined below for objects